isok-agent/src/batch_sender/mod.rs

```diff
@@ -15,16 +15,18 @@ use tonic::transport::Channel;
 #[derive(Debug)]
 pub struct JobResult {
     pub id: JobId,
+    pub name: String,
```

Comment:
Using a `Box<str>` could make it unambiguous that it is in its final, immutable, state, and not a buffer (also getting a slice from a `Box<str>` is `const`).

For small strings or when multiple clones involved I like `ecow::EcoString`.

Reply:
Applied `Box<str>`, didn't use `ecow` to prevent over engineering at this stage of the code